docs/streamtypes/tile-document/api.md

```diff
@@ -30,21 +29,21 @@ const doc = await TileDocument.create(ceramic,
 [:octicons-file-code-16: API reference](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_stream_tile.tiledocument-1.html#create){:target="_blank"}
 
 
-### Parameters
-
-#### ceramic
+#### Parameters
 
+##### ceramic
 
 When creating a TileDocument, the first parameter is the `CeramicAPI` used to communicate with the ceramic node and it is always required. It will either be an instance of [`Ceramic`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_core.ceramic.html){:target="_blank"} when using the Core client or an instance of [`CeramicClient`](https://developers.ceramic.network/reference/typescript/classes/_ceramicnetwork_http_client.ceramicclient.html){:target="_blank"} when using the HTTP client.
 
-#### content
+##### content
 
-The content of your document, in the form of a JSON document. If `schema` is set in the metadata, then the content must conform to the specified schema.
+The content of your TileDocument, in the form of a JSON document. If `schema` is set in the metadata, then the content must conform to the specified schema.
 
-!!! info ""
+!!! warning ""
```

Comment:
why changing this from info to warning?  Feels more like extra information than a warning to me

Reply:
just because the warning color is Ceramic orange, that's the only reason - I've been using warning for all the admonitions

Reply:
Maybe we have another story to change the color of all admonitions to orange that we can later prioritize, since it's spread throughout all the pages